www/include/views/graphs/graphs.html

```diff
@@ -129,6 +129,12 @@
 var templateGraph = Handlebars.compile(source);
 var autoRefresh = false;
 
+// get timezone from localstorage
+// be careful, it seems that it's updated when user logout/login
+const timezone = localStorage.getItem('realTimezone')
+    ? localStorage.getItem('realTimezone')
+    : moment.tz.guess();
```

Comment:
this way you'll get the timezone from the browser. But if the user change this value in the browser, will it be reflected without the need of closing/reopening centreon's tab ?
I mean, shouldn't we use the "ignore cache" option of moment, like in the example from the doc : 
```
moment.tz.guess(); // America/Chicago
// suppose the client's timezone changes to Europe/Berlin
moment.tz.guess(); // America/Chicago
moment.tz.guess(true); // Europe/Berlin
```

Reply:
We don't have a single page application, so I guess it does not change anything to use cache or not for this :)
But anyway, I updated source code to use ignoreCache parameter